packages/nextjs/components/ExtensionCard.tsx

```diff
@@ -0,0 +1,105 @@
+import { useState } from "react";
+import { usePlausible } from "next-plausible";
+import { CheckCircleIcon, DocumentDuplicateIcon } from "@heroicons/react/24/outline";
+import { Address } from "~~/components/scaffold-eth";
+
+type Extension = {
+  name: string;
+  description: string;
+  github: string;
+  installCommand: string;
+  builder: string;
+  coBuilders: string[];
+  youtube?: string;
+};
+
+export const ExtensionCard = ({ extension, isCurated }: { extension: Extension; isCurated: boolean }) => {
+  const [commandCopied, setCommandCopied] = useState(false);
+  const plausible = usePlausible();
+
+  const handleInstallClick = () => {
+    navigator.clipboard.writeText(extension.installCommand);
+    setCommandCopied(true);
```

Comment:
You could use the lib `react-copy-to-clipboard` (already in SE-2)

Reply:
Ups! My bad, fixed in https://github.com/scaffold-eth/scaffoldeth.io/pull/35/commits/ff861f618bcb0e61f65b5cbf2022cd1762efc61b